lazyflow/classifiers/tiktorchLazyflowClassifier.py

```diff
@@ -349,17 +297,18 @@ def predict_probabilities_pixelwise(self, feature_image, roi, axistags=None):
         # select roi from result
         shape_wo_halo = result.shape
         result = result[roiToSlice(*roi)]
-        logger.info(f'Selected roi (start: {roi[0]}, stop: {roi[1]}) from result without halo ({shape_wo_halo}). Now'
-                    f' result has shape: ({result.shape}).')
-
+        logger.info(
+            f"Selected roi (start: {roi[0]}, stop: {roi[1]}) from result without halo ({shape_wo_halo}). Now"
+            f" result has shape: ({result.shape})."
+        )
 
-        self._opReorderAxesOut.AxisOrder.setValue(''.join(axistags.keys()))
-        self._opReorderAxesOut.Input.setValue(vigra.VigraArray(result, axistags=vigra.defaultAxistags('czyx')))
+        self._opReorderAxesOut.AxisOrder.setValue("".join(axistags.keys()))
+        self._opReorderAxesOut.Input.setValue(vigra.VigraArray(result, axistags=vigra.defaultAxistags("czyx")))
         return self._opReorderAxesOut.Output[:].wait()
 
     @property
     def known_classes(self):
-        nr_classes = self._config.get('output_shape')[0]
+        nr_classes = self._config.get("output_shape")[0]
```

Comment:
This will fail if "output_shape" is not set because `None` cannot be indexed. If this is the intended behavior, you can just use an ordinary indexing instead of `get`.

Or did you mean `get("output_shape", 0)`?